reactor-core/src/test/java/reactor/core/publisher/LiftFunctionTest.java

```diff
@@ -69,68 +83,129 @@ void scanOperator(CorePublisher<?> source, int prefetch, Attr.RunStyle runStyle)
 		assertThat(from(liftOperator).scan(Attr.PARENT)).isSameAs(source);
 		assertThat(from(liftOperator).scan(Attr.PREFETCH)).isEqualTo(prefetch);
 		assertThat(from(liftOperator).scan(Attr.RUN_STYLE)).isSameAs(runStyle);
+		assertThat(from(liftOperator).scan(Attr.LIFTER)).isEqualTo(lifterName);
 	}
 
 	@Nested
 	class MonoLiftTest {
 
-		Mono<Integer> source = Mono.just(1).hide();
+		abstract class Base {
```

Comment:
that's what I'm talking about above, that I'd like to avoid. instead we can have 4 tests in the original nested case: `liftScannableMono`/`liftPublisherMono`/`scanScannableMono`/`scanLiftedAsScannable`/`scanLiftedAsPublisher`, wdyt?

Reply:
done